libraries/AP_HAL_ChibiOS/hwdef/common/common_mixf.ld

```diff
@@ -74,6 +74,7 @@ SECTIONS
         *(.ap_romfs*)
         /* moving GCS_MAVLink to ext_flash */
         lib/lib*.a:GCS*.*(.text* .rodata*)
+        lib/lib*.a:tables.*(.text* .rodata*) /* tables for AP_Declination */
```

Comment:
We already have patches for this in master - they're in the libraries themselves rather than hidden in the ld scripts

Reply:
can you please share the link to the PRs?

Reply:
Showed Sid the code on DevCallEU